manifests/v2/third-party/jobset/kustomization.yaml

```
@@ -13,6 +13,8 @@ configMapGenerator:
       disableNameSuffixHash: true
 
 # Add required patches.
-patchesStrategicMerge:
-  - patches/jobset_remove_namespace.yaml # Remove namespace from the JobSet release manifests.
-  - patches/jobset_config_patch.yaml # Add custom manager config to the JobSet.
+patches:
+  # Remove namespace from the JobSet release manifests.
+  - path: patches/jobset_remove_namespace.yaml
```

```suggestion
  - path: patches/jobset_remove_namespace.yaml
    target:
        group: xx
        version: yy
        kind: zz
```

Could we specify the target GVK to make the patch more declaratively?

---

I leaned on the minimal fix, but I agree that'd be better with a target specified. I'm on it.

---

Updated. PTAL.